root/report/ReleasesWithoutCaa.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * @flow strict
+ * Copyright (C) 2023 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import ReleaseList from './components/ReleaseList.js';
+import ReportLayout from './components/ReportLayout.js';
+import type {ReportDataT, ReportReleaseT} from './types.js';
+
+const ReleasesWithoutCaa = ({
+  canBeFiltered,
+  filtered,
+  generated,
+  items,
+  pager,
+}: ReportDataT<ReportReleaseT>): React$Element<typeof ReportLayout> => (
+  <ReportLayout
+    canBeFiltered={canBeFiltered}
+    description={exp.l(
+      `This report shows releases which have no cover art in the Cover Art
+       Archive at all. Most releases have some sort of cover art, so the
+       vast majority of these should eventually get some art
+       {caa_how_to|uploaded to the Cover Art Archive}.
+       Do make sure to only upload images that apply to this specific release
+       (same barcode, format, etc.).`,
```

Comment:
My attempt at improving the wording. Not entirely happy with it but I think it reads a bit better:

```suggestion
      `This report shows releases that have no cover art in the Cover Art
       Archive. Given that most releases have some form of cover art, the
       vast majority of releases in this report should have artwork 
       {caa_how_to|uploaded to the Cover Art Archive}.
       Note that the cover art for a release must always exactly match the
       actual art for that specific release (e.g. same barcode, format, etc.).`,
```

Reply:
We try to avoid `e.g.` (we had a PR recently changing all those to "such as") but in general this seems decent to me. @mwiencek, what do you think?

Reply:
Sounds good to me. :+1: